project3/styles.css

```diff
@@ -0,0 +1,140 @@
+:root {
+    --color-red: #c61a3a;
+    --color-green: #71f79f;
+    --color-offblack: #1e2019;
+    --color-grey: #d1d5de;
+    --color-offwhite: #fafafa;
+    --color-blue: #163346;
+
+}
+
+*,
+*::before,
+*::after {
+    box-sizing: border-box;
+
+}
+
+
+
+body {
+    background-color: var(--color-blue);
+    color: var(--color-offwhite);
+    font-family: 'BenchNine', sans-serif;
+
+
+}
+
+button[type="submit"] {
+    background-color: var(--color-green);
+    display: block;
+    height: 87px;
+    width: 205px;
+    font-family: 'Teko', sans-serif;
+    text-align: center;
+    margin-left: auto;
+    margin-right: auto;
+
+}
+
+
+input[type="radio"],
+input[type="checkbox"] {
+    -moz-appearance: none;
+    -webkit-appearance: none;
+    appearance: none;
+}
+
+
+h2 {
+    font-family: 'BenchNine', sans-serif;
+    font-size: 2em;
+    margin-top: 50px;
+    color: var(--color-blue);
+
+}
+
+h3 {
+    font-family: 'Teko', sans-serif;
+    color: var(--color-offwhite);
+}
+
+legend {
+    display: block;
+    font-family: 'Teko', sans-serif;
+    font-size: 35px;
+    text-transform: uppercase;
+}
+
+fieldset {
+    border: 0;
+    padding: 0;
+}
+
+fieldset span {
```

Comment:
If you want a `span` to be a block element -- and this doesn't change with media queries -- you can just use a `div` and save yourself writing the CSS.